src/axom/inlet/tests/inlet_Inlet.cpp

```diff
@@ -451,6 +451,105 @@ TEST(inlet_Inlet_basic, getNestedValuesAddedUsingTable)
 
 }
 
+TEST(inlet_Inlet, mixLevelTables)
+{
+  // regression test for the lua stack being at the wrong
+  // level with nested tables with non-nested values
+
+  axom::sidre::DataStore dataStore;
+  auto luaReader = std::make_shared<axom::inlet::LuaReader>();
+  luaReader->parseString(
+    "thermal_solver={"
+    "   u0 = { type = \"function\", func = \"BoundaryTemperature\"},"
```

Comment:
Suggestion: You could use the single quote (`'`) as an internal string delimiter in lua.
e.g.
```lua
     "   u0 = { type = 'function', func = 'BoundaryTemperature'}," 
```

Reply:
Done.